Makefile

```diff
@@ -253,3 +253,20 @@ $(GOLANGCI_LINT): ; $(info installing golangci-lint...)
 .PHONY: lint
 lint: | $(GOLANGCI_LINT) ; $(info  running golangci-lint...) @ ## Run golangci-lint
 	$(GOLANGCI_LINT) run --timeout=10m
+
+$(BIN_DIR):
+	@ mkdir -p $(BIN_DIR)
```

Comment:
nit: in other target there is no space between `@` and command. Can we use the same style?

Reply:
sure